docs/Makefile

```diff
@@ -31,7 +31,7 @@ geopmlint:
 	    sphinx-build -M geopmlint source build
 
 clean_all:
```

Comment:
This should go in `.PHONY`, or rename as `clean` which is in `.PHONY` and isn't a build target.